### PR TITLE
Santhosh second commit

### PR DESCRIPTION
**Pull request by santhosh-tunga**
Testing pull request

### CODE REVIEW

**Review by PratishAureole — CHANGES_REQUESTED** [review]
testing change

File: Resx2Xls/CultureInfoComparer.cs [hunk 1]
```diff
@@ -15,6 +15,7 @@ public int Compare(object x, object y)
             if (((x == null) && (y == null)) || x.Equals(y))
             {
                 return 0;  // Santhosh comments
+                // new latest changes
```
Comment (PratishAureole) [inline Resx2Xls/CultureInfoComparer.cs]: This is not proper

File: Resx2Xls/CultureInfoComparer.cs [hunk 2]
```diff
@@ -26,12 +27,12 @@ public int Compare(object x, object y)
             }
             if (!(x is CultureInfo)) //fksjdfjksdkjf
             {
-                throw new ArgumentException("Can only compare Santhosh objects.", "x");
+                throw new ArgumentException("Can", "x");
```
Comment (PratishAureole) [inline Resx2Xls/CultureInfoComparer.cs]: Hi this is test

**Comment by santhosh-tunga** [comment]
Ok i will change

**Review by PratishAureole — CHANGES_REQUESTED** [review]
2nd change request

**Pull request merged**